### PR TITLE
Remove offensive language (Pt. 4)

### PR DESCRIPTION
**Pull request by Holzhaus**
This is part of https://github.com/mixxxdj/mixxx/issues/11931 and contains mostly tooltip and skin changes, as well as replacing some remaining occurrences.

### CODE REVIEW

**Comment by Holzhaus** [comment]
Do we need an alias system for tooltip IDs for backwards-compat with third-party skins? I hope not.

**Comment by ronso0** [comment]
I don't think so. Would be nice to have, but the effort is not worth it IMO. There's a skin warning logged in case of 'invalid' tooltip ids (which needlessly contains the entire file path of legacyskinparser.cpp).
What might be helpful for 3rd-party skin users/developers would be a link to the respective tooltip migration commit (in the changelog maybe?), because, for tooltip ids, we probably won't have something like the 'deprecated, use XX instead' hint in manual > Mixxx controls, it's too much effort and tbh I don't know where to put that so it's actually more discoverable than a link in the changelog.

Whta do you think?

**Comment by ronso0** [comment]
Apart from that the changes LGTM (as good as they can when looking at a diff of 112 files ;), and I'm no checking the skins for obvious regressions.

File: src/track/serato/color.cpp [hunk 1]
```diff
@@ -27,7 +27,7 @@ namespace mixxx {
 /// libraries, we need to convert between the two values.
 ///
 /// See this for details:
-/// https://github.com/Holzhaus/serato-tags/blob/master/docs/colors.md#track-colors
+/// https://github.com/Holzhaus/serato-tags/blob/main/docs/colors.md#track-colors
```
Comment (ronso0) [inline src/track/serato/color.cpp:30]: Btw, wouldn't it be better to move this to a Mixxx repo/branch?

**Review by ronso0 — APPROVED** [review]
Thank you, LGTM

**Pull request merged**